certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW.txt

```diff
@@ -0,0 +1,1651 @@
+[*
```

Comment:
Should this have the .txt extension? I've noticed other files end in just .POW

Reply:
Re uploaded with the '.txt' extension removed.